assets/dev/scss/frontend/_container.scss

```diff
@@ -137,18 +137,17 @@
 		// Since the default is `min-width: auto` for flex items.
 		min-width: 0;
 
-		// Fix iframe-based widgets (ED-5338).
-		// TODO: Find a better solution.
+		// Fix iframe-based widgets (ED-5338) & widget with empty placeholder.
 		&-video,
-		&-google_maps {
-			flex-basis: 100%;
```

Comment:
I have removed `flex-basis`, and replaced it with a `width` of 100%, because it was causing issues when adding custom width to widgets.